syntax = "proto3";

message Input {
    string train_ctf_url = 1;
    string test_ctf_url = 2;
    string query_wl_url = 3;
    string slots_wl_url = 4;
    string intent_wl_url = 5;
    int32 vocab_size = 6;
    int32 num_labels = 7;
    int32 num_intents = 8;
    string sentences_url = 9;
}

message Output {
    string model_url = 1;
    string output_url = 2;
}

service LanguageUnderstanding {
    rpc slot_tagging(Input) returns (Output) {}
    rpc intent(Input) returns (Output) {}
}